file_paths.md

```diff
@@ -305,6 +306,27 @@ The package will install a service named `puppetserver`, create a
         puppetserver/                     # writeable by puppetserver
             puppetserver.pid
 
+# puppetmaster
+
+The package will install a service named `puppetmaster`, create a
+`puppet` user and group, and run the service as the `puppet` user.
+
+    /opt/puppetlabs/server                # serverside apps live underneath
+        data
+            puppetmaster *                # :vardir (and $HOME for services that use it)
```

Comment:
good question. i assume it will have to. this could get awesome. for log and run, the agent package will create /var/run/puppetlabs and /var/log/puppetlabs and puppetmaster will need to depend on the agent.